backend/yarn-error.log

```diff
@@ -1,11 +1,21 @@
 Arguments: 
+<<<<<<< HEAD
```

Comment:
I don't know what this is, but make sure that you delete this so there won't be conflicts